File: android_fragment/app/src/main/java/com/support/generator/module/main/MainActivityContract.java
package com.support.generator.module.main;

import com.support.generator.common.IPresenter;
import com.support.generator.common.IView;

/**
 * @author: leellun
 * @data: 2018/12/24.
 */
public interface MainActivityContract {

  public interface IMainView extends IView {

  }

  public interface IMainPresenter extends IPresenter<IMainView> {

  }
}

File: example/app/src/main/java/gauges/mobile/github/com/mvpapplication/module/main/MainActivityContract.kt
package gauges.mobile.github.com.mvpapplication.module.main

import gauges.mobile.github.com.mvpapplication.common.mvphelper.IPresenter
import gauges.mobile.github.com.mvpapplication.common.mvphelper.IView

/**
 * @author: leellun
 * @data: 2018/12/19.
 *
 */
interface MainActivityContract {

    interface IMainView : IView {

    }

    interface IMainPresenter : IPresenter<IMainView> {

    }
}